app/Services/Validation/SkillDeclarationValidator.php

```diff
@@ -6,9 +6,16 @@
 use App\Models\SkillDeclaration;
 use App\Services\Validation\Rules\PolyExistsRule;
 use App\Services\Validation\Rules\UniqueSkillDeclarationRule;
+use App\Services\Validation\Rules\WordLimitRule;
 
 class SkillDeclarationValidator
 {
+    /**
+     * Maximum amount of words for skill description.
+     *
+     * @var number
+    */
+    const MAXWORDS = 100;
```

Comment:
Could also set this in a constructor to a default value and allow passing in for variable limit